audmath/core/api.py

```diff
@@ -193,6 +194,86 @@ def inverse_normal_distribution(
     return _force_float(x)
 
 
+def rms(
+        x: typing.Union[float, typing.Sequence[float], np.ndarray],
```

Comment:
Input can also be an integer. Should I maybe change to

```suggestion
        x: typing.Union[int, float, typing.Sequence, np.ndarray],
```
?

Reply:
Why do we have to support scalars at all?

Reply:
I would expect that it supports scalars as other numpy functions do it as well, e.g.

```python
>>> np.mean(3)
3.0
```

Reply:
We could use [numpy.typing.ArrayLike](https://numpy.org/devdocs/reference/typing.html#numpy.typing.ArrayLike) as an alternative here for typing, but so far we never used it and had been explicit on the types.

Reply:
> I would expect that it supports scalars as other numpy functions do it as well, e.g.

I guess it depends a bit how generic the library should be. One could argue that we target audio signals here and that we always expect a proper audio signal. An argument for this would be that `rms_db()` does not make much sense otherwise. If we restrict the library to the usage with audio signals, we should also consider proper support for multi-channel input.

Reply:
I added now the arguments `axis` and `keepdims` from `np.mean()` which should be sufficient to handle multi-channel support.
I updated only `rms()`, including the screenshot in the description, to first get some feedback

Reply:
Yes, looks good.

Reply:
Thanks, I updated now `rms_db()` as well.